packages/tkg-clusterclass-aws/bundle/config/upstream/overlay-podsecuritystandard.yaml

```diff
@@ -0,0 +1,115 @@
+            type: string
+            enum: ["", "privileged", "baseline", "restricted"]
+            description: "enforce sets the level for the enforce PodSecurityConfiguration mode. One of \"\", privileged, baseline, restricted."
+            nullable: true
+          enforceVersion:
+            type: string
+            description: "enforceVersion sets the version for the enforce PodSecurityConfiguration mode."
+          audit:
+            type: string
+            enum: ["", "privileged", "baseline", "restricted"]
+            description: "audit sets the level for the audit PodSecurityConfiguration mode. One of \"\", privileged, baseline, restricted."
+            nullable: true
+          auditVersion:
+            type: string
+            description: "auditVersion sets the version for the audit PodSecurityConfiguration mode."
+          warn:
+            type: string
+            enum: ["", "privileged", "baseline", "restricted"]
+            description: "warn sets the level for the warn PodSecurityConfiguration mode. One of \"\", privileged, baseline, restricted."
+            nullable: true
+          warnVersion:
+            type: string
+            description: "warnVersion sets the version for the warn PodSecurityConfiguration mode."
+          exemptions:
+            type: object
+            description: "excemption configuration for the PodSecurityConfiguration."
+            properties:
+              namespaces:
+                type: array
+                items:
+                  type: string
+                description: "namespaces excluded to apply PodSecurityConfiguration Admission."
+  #@overlay/match missing_ok=True
+  patches:
+  #@overlay/remove
+  #@overlay/match by=overlay.map_key("name"),missing_ok=True
+  - name: podSecurityStandard
+  #@overlay/append
+  - name: podSecurityStandard
+    description: "Adds an admission configuration for PodSecurity to the kube-apiserver."
+    definitions:
+    - selector:
+        apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+        kind: KubeadmControlPlaneTemplate
+        matchResources:
+          controlPlane: true
+      jsonPatches:
+      - op: add
+        path: "/spec/template/spec/kubeadmConfigSpec/clusterConfiguration/apiServer/extraArgs/admission-control-config-file"
+        value: "/etc/kubernetes/kube-apiserver-admission-pss.yaml"
+      - op: add
+        path: "/spec/template/spec/kubeadmConfigSpec/clusterConfiguration/apiServer/extraVolumes/-"
+        value:
+          name: admission-pss
+          hostPath: /etc/kubernetes/kube-apiserver-admission-pss.yaml
+          mountPath: /etc/kubernetes/kube-apiserver-admission-pss.yaml
+          readOnly: true
+          pathType: "File"
+      - op: add
+        path: "/spec/template/spec/kubeadmConfigSpec/files/-"
+        valueFrom:
+          template: |-
+            content: |-
+              {{ $namespace_exemptions := printf "%q, %q, %q" "kube-system" "tkg-system" "capz-system" }}
```

Comment:
"capz-system" should be replaced with something else for AWS, I think?

Reply:
resolved, we only have to set it for Azure, but now it will also only get set for azure :-)